src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,39 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={user} />
```

Comment:
let's check if the user exist here
```suggestion
        {user &&  <UserInfo user={user} />}
```